src/EnergyPlus/HybridUnitaryAirConditioners.cc

```diff
@@ -646,7 +646,7 @@ namespace HybridUnitaryAirConditioners {
                 // A19, \ OA requirement pointer
                 ZoneHybridUnitaryAirConditioner(UnitLoop).OARequirementsPtr = UtilityRoutines::FindItemInList(Alphas(19), OARequirements);
                 if (ZoneHybridUnitaryAirConditioner(UnitLoop).OARequirementsPtr == 0) {
-                    ShowSevereError(RoutineName + cCurrentModuleObject + "=\"" + Alphas(1) + " invalid data");
+                    ShowSevereError(RoutineName + cCurrentModuleObject + " = " + Alphas(1) + " invalid data");
                     ShowContinueError("Invalid-not found" + cAlphaFieldNames(19) + "=\"" + Alphas(19) + "\".");

```

Comment:
The object type for the error message is fixed now, but the error message is still basically empty:
```
   ** Severe  ** GetInputZoneHybridUnitaryAirConditioners: ZoneHVAC:HybridUnitaryHVAC = MUNTERSEPX5000 invalid data
   **   ~~~   ** Invalid-not found="".
```
In the call to `getObjectItem`, the field names are returned in `cAlphaFields`. Some error messages use this variable, but a few (like the error shown here) are using `cAlphaFieldNames` which is blank. Replace all of these with `cAlphaFields` and the error message should show the field name that's in error.